Blizzard.IO.RabbitMQ/Entities/RabbitExchange.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Blizzard.IO.RabbitMQ.Entities
+{
+    public class RabbitExchange
+    {
+        public string Name { get; set; }
+        public RabbitExchangeType Type { get; set; }
```

Comment:
I think that this property should be the actual `string` type (etc: "fanout", "topic") instead of using an enumeration
It is more strait-forward, and you wouldn't have to edit this enum in the future when adding new types

Reply:
Im trying to protect the user from tyring errors cause it is just string and the compiler cant tell if it is a valid type or not.
Any way you that in the near future there will be more exchange types?